index.js

```diff
@@ -10,7 +10,22 @@ class Tree {
     this.root = node;
   }
 
-  findNode(data) {}
+  findNode(data) {
+    let currentNode = this.root;
+    let queue = [this.root];
+    while(queue.length){
```

Comment:
what would happen if used `queue.pop()` instead of .shift?

How would that change the search? 

How would it affect the Big O?